packages/siopv2-oid4vp-op-auth/src/services/Siopv2MachineService.ts

```diff
@@ -153,16 +158,115 @@ export const siopSendAuthorizationResponse = async (
 
     idOpts = presentationsAndDefs[0].idOpts
     presentationSubmission = presentationsAndDefs[0].presentationSubmission
+
+    logger.log(`Definitions and locations:`, JSON.stringify(presentationsAndDefs?.[0]?.verifiablePresentations, null, 2))
+    logger.log(`Presentation Submission:`, JSON.stringify(presentationSubmission, null, 2))
+    const mergedVerifiablePresentations = presentationsAndDefs?.flatMap((pd) => pd.verifiablePresentations) || []
+    return await session.sendAuthorizationResponse({
+      ...(presentationsAndDefs && { verifiablePresentations: mergedVerifiablePresentations }),
+      ...(presentationSubmission && { presentationSubmission }),
+      // todo: Change issuer value in case we do not use identifier. Use key.meta.jwkThumbprint then
+      responseSignerOpts: idOpts!,
+    })
+  } else if (request.dcqlQuery) {
```

Comment:
Integration attempts were supposed to be removed from this PR.
We haven no code to create a presentation from DCQL (see SPRIND-150)